Cesium.CodeGen/Ir/Types/StructType.cs

```diff
@@ -46,5 +46,5 @@ public TypeDefinition Emit(string name, TranslationUnitContext context)
     public TypeReference Resolve(TranslationUnitContext context) =>
         context.GetTypeReference(this) ?? throw new CompilationException($"Type {this} was not found.");
 
-    public int SizeInBytes => throw new WipException(232, $"Could not calculate size for {this} yet.");
+    public int SizeInBytes => Members.Sum(_ => _.Type.SizeInBytes);
```

Comment:
But now wait a minute. The actual memory layout of a `struct` includes a lot of various paddings. I don't believe this is correct even for simple cases like `struct Foo { int a; char b; }`